Forms/DrawForm/DrawForm.cs

```diff
@@ -34,10 +35,17 @@ public partial class DrawForm : Form
 
         private Pen pen;
 
+        private ToolPanel toolPanel;
+
+        private WeightTable weightTable;
+
+        private MatrixWeightPanel matrixWeightPanel;
+
         private Arrow arrow;
 
+        private List<List<InputCountBox>> matrix;
```

Comment:
Why do you create the "private List<List<Input Count Box>> matrix" field if it is not used anywhere except in the constructor, where it is only initialized?